bindata/etcd/pod.yaml

```diff
@@ -335,3 +335,9 @@ ${COMPUTED_ENV_VARS}
     - hostPath:
         path: /var/log/etcd
       name: log-dir
+    - hostPath:
+        path: /var/backup/etcd
+      name: backup-dir
+    - hostPath:
+        path: /etc/kubernetes
+      name: config-dir
```

Comment:
Still have to review to this later and see if all of this aligns with what's outlined in the enhancement, but manifest changes related to this feature like this need to be feature gated and can't be in GA by default.